packages/fxa-auth-server/package.json

```diff
@@ -33,7 +33,7 @@
     "maildev-preview": "ts-node ./lib/senders/emails/preview-in-maildev.ts",
     "start-mjml": "NODE_ENV=dev ts-node ./bin/mjml-server.ts",
     "storybook": "npm run emails-scss && start-storybook -p 6010 --no-version-updates",
-    "storybook-mjml": "npm-run-all -p start-mjml storybook"
+    "storybook-mjml": "nodemon -e scss,ts  --exec \"npm-run-all -p start-mjml storybook\""
```

Comment:
Still hoping to find a better solution for this.. will follow up shortly

Reply:
FWIW if we don't find a better solution, I think this is fine for now if it covers the basis we need it for just to make developing these templates for at least a few more days a bit less painful, we can maybe even revisit in a few weeks.